xml/runtime_config.xml

```diff
@@ -10,60 +10,259 @@
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
    <dm:translation>yes</dm:translation>
-   <dm:release>SES 6</dm:release>
+   <dm:release>SES 7</dm:release>
   </dm:docmanager>
  </info>
  <para>
-  This chapter provides a list of important &ceph; cluster settings and their
-  description. The settings are sorted by topic.
+  This chapter describes how to configure the &ceph; cluster by means of
+  configuration options.
  </para>
- <sect1 xml:id="ceph-config-runtime">
-  <title>Runtime Configuration</title>
+ <sect1 xml:id="cha-ceph-configuration-ceph-conf">
+  <title>The <filename>ceph.conf</filename> File</title>
 
   <para>
-   The actual cluster behavior is determined not by the current state of the
-   <filename>ceph.conf</filename> file but by the configuration of the running
-   &ceph; daemons, which is stored in memory.
+   &cephadm; uses a minimized <filename>ceph.conf</filename> file that only
+   contains a minimal set of options for connecting to MONs, authenticating,
+   and fetching configuration information. In most cases this is limited to the
+   <option>mon_host</option> option (although this can be avoided through the
+   use of DNS SRV records).
   </para>
 
-  <para>
-   You can query an individual &ceph; daemon for a particular configuration
-   setting using the <emphasis>admin socket</emphasis> on the node where the
-   daemon is running. For example, the following command gets the value of the
-   <option>osd_max_write_size</option> configuration parameter from the daemon
-   named <literal>osd.0</literal>:
-  </para>
-
-<screen>&prompt.cephuser;ceph --admin-daemon /var/run/ceph/ceph-osd.0.asok \
-config get osd_max_write_size
-{
-  "osd_max_write_size": "90"
-}</screen>
+  <important>
+   <para>
+    The <filename>ceph.conf</filename> file no longer serves as a central place
+    for storing cluster configuration in favor of the configuration database
+    (see <xref linkend="cha-ceph-configuration-db"/>).
+   </para>
+   <para>
+    If you still need to change cluster configuration via the
+    <filename>ceph.conf</filename> file, for example, because you use a client
+    that does not support reading options form the configuration database, you
```

Comment:
```suggestion
    <filename>ceph.conf</filename> file&mdash;for example, because you use a client
    that does not support reading options form the configuration database&mdash;you
```